ddl/BUILD.bazel

```diff
@@ -217,6 +217,7 @@ go_test(
         "@com_github_pingcap_failpoint//:failpoint",
         "@com_github_pingcap_kvproto//pkg/metapb",
         "@com_github_pingcap_log//:log",
+        "@com_github_stretchr_testify//assert",
```

Comment:
Perhaps we don't need this dependency?

Reply:
I think Someone use testify/assert in the ddl test.

Reply:
It is used in the ```ddl/cancel_test.go```

Reply:
I find it is useless. so I have removed all ```testify/assert```